feat: ja translate develop/datatypes

riywo opened the pull request:
Close #184

riywo committed:
feat: ja translate develop/datatypes

riywo commented:
@poppoerika Check this out!

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/datatypes.md:
```diff
@@ -1,28 +1,31 @@
-# Supported data types in Momento Cache and Momento Topics
-Momento Cache provides a group of core data types to match up with common types in various modern programming languages. This page has a list of the core data structures and information about each of them.
+# Momento Cache と Momento Topics でサポートしているデータ型
+Momento Cache は、現代的な様々なプログラミング言語に共通する型と対応する、コアとなるデータ型を提供しています。このページでは、コアのデータ構造の一覧と、それぞれの情報を紹介しています。
```
Do we need `、` here?
`現代的な様々なプログラミング言語に共通する型と対応する、`

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/datatypes.md:
```diff
@@ -1,28 +1,31 @@
+保存される全てのデータはバイト配列で表現されます。Momento SDK は、生のバイトやUTF-8 の文字列として皆さんのデータを保存するために簡単に使えるメソッドを提供してます。これらは、スカラーデータ (文字列、キャラクタ、数字、等) か、バイナリデータ (画像、Protocol Buffer のシリアライズされた配列) の形になります。
 
-<img src="/img/collection_data_types.png" alt="Collection data types drawing | Momento Cache" width="80%"/>
+### 集合データ型 (CDT)
```
I think we can use `コレクション` here

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/datatypes.md:
```diff
@@ -1,28 +1,31 @@
 
-#### Lists
-A list is a collection of ordered elements, sorted in the sequence each element was inserted.
+集合データ型は、関連するデータを一つの項目にグループ化したものです。それらはバイト配列の値として保存され、Momento SDK ではそれぞれに API が提供されています。例えば、DictionaryFetch、ListFetch、SetFetch 等です。
```
`集合データ型は`　-> `コレクションデータ型`

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/datatypes.md:
```diff
@@ -1,28 +1,31 @@
-A list is a collection of ordered elements, sorted in the sequence each element was inserted.
+集合データ型は、関連するデータを一つの項目にグループ化したものです。それらはバイト配列の値として保存され、Momento SDK ではそれぞれに API が提供されています。例えば、DictionaryFetch、ListFetch、SetFetch 等です。
+
+![集合データ型 | Momento Cache](/img/collection_data_types.png)
```
`[集合データ型`　-> `コレクションデータ型`

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/datatypes.md:
````diff
@@ -39,12 +42,13 @@ A list is a collection of ordered elements, sorted in the sequence each element
 ```
 
-To learn the API methods, check the [API reference for list collection data types](./api-reference/list-collections.md).
+API メソッドは、[リスト集合データ型の API リファレンス](./api-reference/list-collections.md)をご覧下さい。
````
same comment as https://github.com/momentohq/public-dev-docs/pull/194/files#r1186332378

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/datatypes.md:
````diff
@@ -54,10 +58,11 @@ An example of using a dictionary is when you need to store data together and ret
 ```
 
-To learn the API methods, check the [API reference for dictionary collection data types](./api-reference/dictionary-collections.md).
+API メソッドは、[辞書集合データ型の API リファレンス](./api-reference/dictionary-collections.md)をご覧下さい。
````
same comment as https://github.com/momentohq/public-dev-docs/pull/194/files#r1186332378

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/datatypes.md:
````diff
@@ -70,11 +75,11 @@ A set is an unordered collection of unique elements, each in string format. For
 ```
 
-To learn the API methods, check the [API reference for set collection data types](./api-reference/set-collections.md).
+API メソッドは、[セット集合データ型の API リファレンス](./api-reference/set-collections.md)をご覧下さい。
````
same comment as https://github.com/momentohq/public-dev-docs/pull/194/files#r1186332378

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/datatypes.md:
```diff
@@ -91,29 +96,29 @@ A sorted set is a collection of unique elements with a value (string) and score
 :::
 
-To learn the API methods, check the [API reference for sorted set collection data types](./api-reference/sorted-set-collections.md).
+API メソッドは [ソート済セット集合データ型の API リファレンス](./api-reference/sorted-set-collections.md)をご覧下さい。
```
same comment as https://github.com/momentohq/public-dev-docs/pull/194/files#r1186332378

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/datatypes.md:
```diff
@@ -91,29 +96,29 @@ A sorted set is a collection of unique elements with a value (string) and score
 <details>
-  <summary>Does Momento Cache store nested data in collection data types?</summary>
-Not directly. Your best option is to store this data as a JSON object and then use your favorite JSON library to serialize the JSON document into a byte array and insert that byte array into Momento Cache.
+  <summary>Momento Cache は集合データ型でネストされたデータを保存できますか？</summary>
```
same comment as https://github.com/momentohq/public-dev-docs/pull/194/files#r1186332378

riywo commented:
@poppoerika Addressed your comments! https://github.com/momentohq/public-dev-docs/compare/7e87d6b0413bd897499f73e7069aad6102645030..561e10dfe2a56341a7c641da5f04db1bf47c5617
